src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v1/switches/RulesSyncResult.java

```diff
@@ -39,4 +40,9 @@ public RulesSyncResult(
 
         this.installedRules = installedRules;
     }
+
+    @JsonGetter("installed-rules-hex")
```

Comment:
Other fields here are underscore separated, this one is dash-separated.

Reply:
in some day at past all agreed to use dash instead of underscores for any new fields in api